app/models/ability.rb

```diff
@@ -48,11 +48,13 @@ def caseworker_abilities
     can_search
     can_manage_links
     can_manage_self
+    can_show_defendant
```

Comment:
I just extended the `can_search` method as it all just relates

see this [commit that i applied for hearing search](https://github.com/ministryofjustice/laa-court-data-ui/pull/217/commits/b88e090288f6df83e629d7b11052b5d4602c4e77)

Reply:
👍 Added can %i[show], CourtDataAdaptor::Resource::Defendant to can_search